src/main/java/com/denizenscript/depenizen/bukkit/objects/superiorskyblock/SuperiorSkyblockIslandTag.java

```diff
@@ -0,0 +1,386 @@
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name balance
+        // @input ElementTag(Decimal)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes an island's bank balance.
+        // @tags
+        // <SuperiorSkyblockIslandTag.balance>
+        // -->
+        tagProcessor.registerMechanism("balance", false, ElementTag.class, (object, mechanism, value) -> {
+            if (mechanism.requireDouble()) {
+                object.getIsland().getIslandBank().setBalance(mechanism.getValue().asBigDecimal());
+            }
+        });
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name description
+        // @input ElementTag
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes an island's description. Leave blank to remove.
+        // @tags
+        // <SuperiorSkyblockIslandTag.description>
+        // -->
+        tagProcessor.registerMechanism("description", false, ElementTag.class, (object, mechanism, value) -> {
+            object.getIsland().setDescription(mechanism.getValue().asString());
+        });
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name locked
+        // @input ElementTag(Boolean)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes whether an island is locked to visitors.
+        // @tags
+        // <SuperiorSkyblockIslandTag.locked>
+        // -->
+        tagProcessor.registerMechanism("locked", false, ElementTag.class, (object, mechanism, value) -> {
+            if (mechanism.requireBoolean()) {
+                object.getIsland().setLocked(mechanism.getValue().asBoolean());
+            }
+        });
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name name
+        // @input ElementTag
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes an island's name.
+        // @tags
+        // <SuperiorSkyblockIslandTag.name>
+        // -->
+        tagProcessor.registerMechanism("name", false, ElementTag.class, (object, mechanism, value) -> {
+            if (!mechanism.getValue().asString().isEmpty() && SuperiorSkyblockAPI.getIsland(mechanism.getValue().asString()) == null) {
+                object.getIsland().setName(mechanism.getValue().toString());
+            }
+        });
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name size
+        // @input ElementTag(Number)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes an island's border size.
+        // @tags
+        // <SuperiorSkyblockIslandTag.size>
+        // -->
+        tagProcessor.registerMechanism("size", false, ElementTag.class, (object, mechanism, value) -> {
+            if (mechanism.requireInteger() && mechanism.getValue().asInt() >= 1) {
```

Comment:
this `>= 1` check needs its own else with an echoError (so it doesn't just silently ignore incorrect inputs)